tools/lib/logreader.py

```diff
@@ -140,6 +140,7 @@ def internal_source(sr: SegmentRange, mode: ReadMode, file_ext: str = "bz2") ->
   def get_internal_url(sr: SegmentRange, seg, file):
     return f"cd:/{sr.dongle_id}/{sr.log_id}/{seg}/{file}.{file_ext}"
 
+  # TODO: list instead of using static URLs to support routes with multiple file extensions
```

Comment:
forgot to say why we can't now: I believe some code relies on implicitly downloading all rlogs to the office with the HEAD requests we do